config.yml.erb

```diff
@@ -206,11 +206,6 @@ freegeoip_host:
 geocoder_redis_url:
 solr_server:
 
-# Asynchronous queue-processor
-pd_workshop_queue_url:
-activity_queue_url:
-process_queues:
```

Comment:
There's currently a chicken/egg issue with removing properties that have residual Chef-attribute configuration- as soon as the property is removed from the default config here, it'll cause an error on startup on the next CI build until the property is removed from Chef / `globals.yml` as well. We can fix this in one of two ways:

1. Leave these properties in for now, deploy the rest of the PR (removing all references), then remove the properties from Chef, then merge the property-removal in a followup PR;
2. I can commit a change to the config-loader that will warn (instead of error out) when unexpected/undefined properties are present in `globals.yml`, which will make this more seamless.

Reply:
@wjordan I'm happy to do option 1 if that works for you.

Reply:
Maybe worth also mentioning option 3 more explicitly:
3. Merge this PR as-is, expect errors on the initial CI build in environments where those properties are set, and manually remove them (both from the hosted Chef-attributes and from `globals.yml`) from each environment at that point (and possibly manually re-start the build).

Going ahead with option 1 sounds good though!

Reply:
I've now updated this PR for option 1.